src/civetweb.c

```diff
@@ -1039,6 +1046,52 @@ static int mg_snprintf(struct mg_connection *conn, char *buf, size_t buflen,
     return n;
 }
 
+
```

Comment:
general comment for readability: split long lines at appropriate boundaries
eg. "for ( long line; possibly more than; 80 chars)" becomes
for (long line;
       possibly more than;
       80 chars)
eg. "func_call(with, a, long, arg, list)" becomes
func_call(with, a,
                long, arg,
                list)